### PR TITLE
bump 1.1.1

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
ping @dongluochen @abronan @nishanttotla

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.1.1 (2016-02-17)
+
+- Performence improvements around networking with 1.10
```
Comment (dongluochen) [inline CHANGELOG.md]: `1.10` -> `Docker 1.10`

Comment (nishanttotla) [inline CHANGELOG.md]: Typo: `Performence` -> `Performance`

Comment (vieux) [inline CHANGELOG.md]: thanks

**Comment by nishanttotla** [comment]
LGTM

**Comment by dongluochen** [comment]
LGTM

**Pull request merged**